Platform.PCL/Realm.PCL/Schema/RealmSchemaPCL.cs

```diff
@@ -38,7 +38,14 @@ public class RealmSchema : IReadOnlyCollection<ObjectSchema>
         /// Gets the number of known classes in the schema.
         /// </summary>
         /// <value>The number of known classes.</value>
-        public int Count { get; }
```

Comment:
The only constructor of this class is private and throws. Is there a chance that the `Count` property would ever be accessed?